Test/Invoke-WhiskeyTask.Tests.ps1

```diff
@@ -683,6 +683,24 @@ Describe 'Invoke-WhiskeyTask.when OnlyOnBranch and ExceptOnBranch properties are
     ThenTaskNotRun 'Invoke-WhiskeyPowerShell'
 }
 
+Describe 'Invoke-WhiskeyTask.when WorkingDirectory property is defined' {

```

Comment:
Add an assertion to both of these that `Invoke-WhiskeyTask` actually moves to the working directory and pops out of it. Unless its already doing it?

Reply:
Added some tests for this.